tools/detect_unused_modules

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh -e
 
 for test in $(git ls-files "*tests/*.pm") ; do
-    t=$(echo $test | sed -e 's@tests/.*/@@' -e 's@.pm@@')
+    t=$(echo $test | sed -e 's@tests/.*/@@' -e 's@.pm$@@')
```

Comment:
I guess that should be `s@\.pm$@@` actually to match only on real dots, not any character. Can you change that and check please?

Reply:
yes, the patch will be ready tomorrow. :-)